esmvalcore/config-developer.yml

```diff
@@ -194,8 +195,12 @@ native6:
   cmor_strict: false
   input_dir:
     default: 'Tier{tier}/{dataset}/{latestversion}/{frequency}/{short_name}'
+    IPSL: '{root}/{account}/{model}/{status}/{exp}/{simulation}/{dir}/{freq}'
   input_file:
     default: '*.nc'
+    IPSL:
+      - '{simulation}_*_{ipsl_varname}.nc'
+      - '{simulation}_*_{group}.nc'
```

Comment:
Is this correct or a left-over from the previous attempt to treat the model data as native6?